umgc/fall2021.mesmerize
# V0.1 - Initial Commit

This is the initial commit that conforms to the design documents, with many more features to come!

Testing instructions:

(Make sure you download the API-29 Base Image SDK on the Google Nexus emulator)

You should already have flutter installed. Pull the repo, pull this branch, go to the folder, start your emulator, type flutter pub get, then type flutter run after the emulator has already started. 

Acceptance Criteria:

App loads on the Android Emulator
iOS compatibility not ensured at the moment.
